src/lib/middleware/response-time-metrics.ts

```diff
@@ -1,10 +1,10 @@
-'use strict';
+import  * as responseTime from 'response-time';
+import { REQUEST_TIME } from '../events';
 
-const responseTime = require('response-time');
-const { REQUEST_TIME } = require('../events');
+var _responseTime = responseTime.default
```

Comment:
It wasn't clear to me why I needed to pull out the 'default' for `responseTime` to work. But if I didn't the import came in as 'undefined'.

Reply:
did you try `import  responseTime from 'response-time';`?

Reply:
my suggestion does not work.. 

But we should consider not using default exports yet, as it also can lead to poorer refactoring capabilities. Not using default exports works just fine. 

ref:
https://basarat.gitbook.io/typescript/main-1/defaultisbad